### PR TITLE
Feat(CWCP-28): View Cart Items

### PR DESCRIPTION
**Pull request by Sprytte**
JIRA: https://champlainsaintlambert.atlassian.net/browse/CWCP-28

Context:
This ticket is to allow the user to view the items that they added in their shopping cart.

Changes
- ShoppingCart page
- CSS styling
- Shopping cart icon in header.js

Before and After UI (Required for UI-impacting PRs)

Before:
image

After:
image
image

### CODE REVIEW

File: src/Pages/ShoppingCart.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                        </thead>
+                        <tbody>
+                            {cart.cartItems && cart.cartItems.map(product => {
+                                    return <tr> 
+                                            <td><img src={product.imgURL} width={70} height={60} alt='product default image'/> </td>
+                                            <td>{product.name}</td>
+                                            <td>${product.unitPrice} CAD</td>
+                                            <td>- &nbsp;{product.quantity} &nbsp; +</td>
+                                            <td>${product.totalPrice} CAD</td>
+                                        </tr>
```
Comment (ayasophya) [inline src/Pages/ShoppingCart.js:50]: good job figuring out that td is in tr!!

**Review by ayasophya — APPROVED** [review]
nice one again emly!!

File: src/Pages/ShoppingCart.js (hunk 1 above)
Comment (Sprytte) [reply src/Pages/ShoppingCart.js:50]: first of all, never write my name that way again. Second of all, I told you that in confidence

**Review by MinseokBUZZ — APPROVED** [review]
looks good, Great Job on implementing View Car Items!

**Pull request merged**